src/test/regress/sql/multi_mx_schema_support.sql

```diff
@@ -326,7 +328,8 @@ ALTER TABLE mx_old_schema.table_set_schema SET SCHEMA mx_new_schema;
 
 SELECT objid::oid::regnamespace as "Distributed Schemas"
     FROM pg_catalog.pg_dist_object
-    WHERE objid::oid::regnamespace IN ('mx_old_schema', 'mx_new_schema');
+    WHERE objid::oid::regnamespace IN ('mx_old_schema', 'mx_new_schema')
+    ORDER BY "Distributed Schemas";
```

Comment:
Should we order by `"Distributed Schemas"::text` because it sounds like safer to compare their names, not their OIDs?

Reply:
Because we cast them to regnamespace I'm pretty sure it's already ordering them by display name instead of actual raw oid value.

Reply:
I think this is not what happens when you order by regnamespace:

```sql
select oid, nspname from pg_namespace order by oid::regnamespace;
┌───────┬────────────────────┐
│  oid  │      nspname       │
├───────┼────────────────────┤
│    11 │ pg_catalog         │
│    99 │ pg_toast           │
│ 13157 │ information_schema │
│ 52422 │ public             │
│ 52424 │ columnar_internal  │
│ 52454 │ columnar           │
│ 52480 │ citus              │
│ 52795 │ citus_internal     │
└───────┴────────────────────┘
(8 rows)
```

Reply:
Interesting, fair enough then. Changed the test.